api/src/services/checkouts/checkouts.js

```diff
@@ -44,9 +42,9 @@ export const createStripeCheckoutSession = async ({
   return session
 }
 
-export const redirectToStripeCheckout = async url => {
-  // probably best this logic lives in an serverless function
-}
+// export const redirectToStripeCheckout = async url => {
+//   // probably best this logic lives in an serverless function
+// }
```

Comment:
Let's not export this one if it's a noop

Reply:
Fair enough